commonui/src/main/java/com/hyperwallet/android/ui/common/view/error/DefaultErrorDialogFragment.java

```diff
@@ -73,7 +77,11 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     @Override
     public void onCancel(DialogInterface dialog) {
         super.onCancel(dialog);
-
+        ArrayList<Error> errors = getErrors();
+        if(errors.get(0).getCode().equals(ErrorTypes.AUTH_TOKEN_ERROR)){
```

Comment:
lgtm overall just add space after `if` then this PR is good to me

Reply:
done